### PR TITLE
Feature/aba/by jason/profile reactive form

### PR DESCRIPTION
**Pull request by jqj5405**
@ardithoxha @SuchenShi code review

### CODE REVIEW

**Comment by ardithoxha** [comment]
Jason, I reviewed  your code and the profile page looks great. The fields are editable, the save button gets the new inputs and the error descriptions come up when necessary. This looks ready to be merged to me.

File: TeamProject/src/app/modules/profile/profile-page/profile-page.component.html [hunk 1]
```diff
@@ -1,45 +1,104 @@
...
+              Password can't be blank
+            </div>
+            <div [hidden]="!passwordFC.errors.minlength">
+              Min length (5) required
+            </div>
+            <div [hidden]="!passwordFC.errors.pattern">
+              (1) uppercase and (1) special character (#?!@$%^&*) required
+            </div>
+          </mat-error>
+          <mat-hint>Min length (5); Use (1) uppercase and (1) special character</mat-hint>
```
Comment (SuchenShi) [inline TeamProject/src/app/modules/profile/profile-page/profile-page.component.html:80]: I am just wondering if there is a better presentation for indicating the number of uppercase or special characters required. (1) is just a bit confusing.

**Comment by SuchenShi** [comment]
This looks great to me. Go ahead and merge it.

File: TeamProject/src/app/modules/profile/profile-page/profile-page.component.html (hunk 1 above)
Comment (jqj5405) [reply TeamProject/src/app/modules/profile/profile-page/profile-page.component.html:80]: What do you suggest?

**Pull request merged**